src/containerapp/azext_containerapp/_params.py

```diff
@@ -462,9 +462,9 @@ def load_arguments(self, _):
         c.argument('polling_interval', type=int, help="Interval to check each event source in seconds. Defaults to 30s.", default=30)
 
     with self.argument_context('containerapp job create') as c:
-        c.argument('system_assigned', help='System assigned identity.', action='store_true')
+        c.argument('system_assigned', help='Boolean indicating whether to assign system-assigned identity.', action='store_true')
         c.argument('trigger_type', help='Trigger type. Schedule | Event | Manual')
-        c.argument('user_assigned', help='User assigned identity.')
+        c.argument('user_assigned', nargs='+', help='Space-separated user identities to be assigned.')
```

Comment:
Could you refer to the guideline [managed_identity_command_guideline.md](https://github.com/Azure/azure-cli/blob/dev/doc/managed_identity_command_guideline.md) and replace the original `--system-assigned` and `--user-assigned` parameters with `--mi-system-assigned` and `--mi-user-assigned` parameters? The original parameters do not need to be removed directly, they can be hidden first

Reply:
@zhoxing-ms can you give more context on this `The original parameters do not need to be removed directly, they can be hidden first`. Does this mean that we should rename the identity params here, adding "mi-" to them, without updating the actual implementation in custom.py? That would give linter validation errors right?

Reply:
Yes, I left some code suggestions, please take a look~